aviary/interface/methods_for_level2.py

```diff
@@ -433,6 +432,44 @@ def _update_metadata_from_subsystems(self):
             if variable in self.meta_data:
                 self.meta_data.pop(variable)
 
+    def phase_separator(self):
+        """
+        This method checks for reserve=True & False
+        Returns errors if reserve = False no reserve no specified after reserve = True 
+        return two dictionaries of phases, regular_phases and reserve_phaes
+        """
```

Comment:
What about a case where we have actual landing phases defined. In such a case, we probably want to "branch" the reserve mission phases so that they connect to the final descent phase before the landing phases.

Reply:
We don't have any sense of connecting anything but mass & distance between the regular phases and the reserve phases. I think we'd need a follow-on PR do start dealing with / letting the user specify tandem phases.

Reply:
I don't want to focus on enabling full tandem phases for this PR. That's out of scope for this job.